pkg/client/github.go

```diff
@@ -130,6 +130,7 @@ type GithubTeam struct {
 	ID           api.ID
 	Organisation string
 	Name         string
+	TeamID       int64
```

Comment:
Just double checking: Is team ID actually an int and not a string? If team ID starts with zero, and is converted to an int, and then back to a string, the zero will be lost.

Reply:
Yepp, its actually an int :) 
See
go-github/v32 ...teams.go

```
// Team represents a team within a GitHub organization. Teams are used to
// manage access to an organization's repositories.
type Team struct {
	ID          *int64  `json:"id,omitempty"`
```

Reply:
And yepp, the sync will come later . Waited with that since most people in Origo do not have a public facing e-mail on their git-hub account (yet).